AdyenActions/AdyenActionComponent.swift

```diff
@@ -204,15 +204,17 @@ public final class AdyenActionComponent: ActionComponent, ActionHandlingComponen
     }
     
     private func handle(_ action: QRCodeAction) {
-        let component = QRCodeComponent(context: context)
-        component.configuration.style = configuration.style.qrCodeComponentStyle
-        component._isDropIn = _isDropIn
-        component.delegate = delegate
-        component.presentationDelegate = presentationDelegate
-        component.configuration.localizationParameters = configuration.localizationParameters
-        
-        component.handle(action)
-        currentActionComponent = component
+        if action.paymentMethodType == .pix {
```

Comment:
1) this could be a `guard`
2) not sure this is good idea - If I am not mistaken - we have other `qrCode` action(s) 🤔